src/add-remove.js

```diff
@@ -1,3 +1,5 @@
+import cleanListener from './clear.js';
+
```

Comment:
- [ ] On line 17:

![image](https://user-images.githubusercontent.com/57421823/165696691-20be4f41-9d44-48d5-8875-65efc6953822.png)

There is a line of old commented code.  Please, kindly remove it to improve the cleanliness of the repo 💪